sampleZone.c

```diff
@@ -135,6 +139,22 @@ int main(int argc, char *argv[]) {
     int selectedFileIndex = 0;
     audioFile *selectedFile = &files[selectedFileIndex];
 
+    //Check files for errors (TODO: More robust error checking code. Make this not broken)
```

Comment:
This is step 2 of the process. My PR will fix the bug so the program doesn't break, and I'm going to improve the feature by properly listing the broken files on load.